test/hw7/test_task2.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+from All_home_works.hw7.task2 import backspace_compare
+
+
+@pytest.mark.parametrize(  # noqa: PT006
```

Comment:
What is PT006?

Reply:
https://github.com/E1-Bart0/homework-repo/pull/25#discussion_r627397514